directory_header_footer/context_processors.py

```diff
@@ -0,0 +1,23 @@
+from django.conf import settings
+
+
+def sso_processor(request):
+    url = request.build_absolute_uri()
+    sso_password_reset_url = getattr(settings, 'SSO_PASSWORD_RESET_URL', None)
+    return {
+        'sso_password_reset_url': sso_password_reset_url,
+        'sso_user': request.sso_user,
+        'sso_is_logged_in': request.sso_user is not None,
+        'sso_login_url': '{0}?next={1}'.format(settings.SSO_LOGIN_URL, url),
+        'sso_register_url': settings.SSO_SIGNUP_URL,
+        'sso_logout_url': settings.SSO_LOGOUT_URL,
+        'sso_profile_url': settings.SSO_PROFILE_URL,
+    }
+
+
+def header_footer_context_processor(request):
+    active_classes = getattr(settings, 'HEADER_FOOTER_CSS_ACTIVE_CLASSES', {})
```

Comment:
```
<a class="{% if HEADER_FOOTER_CSS_ACTIVE_CLASSES.fab %}navigation-main-link--active{% endif %}">Find a buyer</a>```

if `HEADER_FOOTER_CSS_ACTIVE_CLASSES` is {} then the template code will evaluate to false and not use the active class.